config/crds/kudo_v1alpha1_teststep.yaml

```diff
@@ -32,6 +64,11 @@ spec:
             object represents. Servers may infer this from the endpoint the client
             submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/api-conventions.md#types-kinds'
           type: string
+        kubectl:
```

Comment:
also added because `make manifests` was not run, it should have already been part of different PR that introduced this field